docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -124,3 +124,22 @@ docker run -it -p 443:443 \
   -e EPR_TLS_CERT=/etc/ssl/package-registry.crt \
   docker.elastic.co/package-registry/distribution:{version}
 ----
+
+[discrete]
+[[air-gapped-limitations]]
+== Limitations for Agent Upgrades
+
+The [[upgrade-elastic-agent]] feature in technical preview does not currently support network restricted environments without access to artifacts.elastic.co.
+To upgrade {agent}s:
+
+1. Download the new version from the https://www.elastic.co/downloads/elastic-agent[download page]
+2. Place the tarball on a secure internal network location
+3. Use the <<elastic-agent-upgrade-command,`upgrade`>> command with a custom `--source-uri` option
+
+[discrete]
+=== Example
+
+["source", "sh"]
```

Comment:
You need `subs="attributes"` here, too, if you want `{version}` to resolve to a value. It will resolve to the current version, though, which might be a little weird. So maybe you want to leave this as-is.

Reply:
yah actually that makes sense. Then you should change line 142 to say `["source", "sh", subs="attributes"]` (my thinking was a bit backwards yesterday for some reason).